docs/site/content/docs/latest/vsphere.md

```diff
@@ -10,9 +10,8 @@ This topic explains how to prepare your environment before you deploy a manageme
 * [ ] Ensure the Tanzu CLI is installed locally on the bootstrap machine. See [Install the Tanzu CLI](cli-installation.md).
 * [ ] Review the vSphere account reference information here: [Reference for vSphere account](ref-vsphere.md)
 * [ ] Ensure that vSphere meets the following general requirements:
-  * [ ] A vSphere 7, vSphere 6.7u3, VMware Cloud on AWS, or Azure VMware Solution account with:
-    * [ ] **vSphere 6.7**: an Enterprise Plus license
-    * [ ] **vSphere 7**
+  * [ ] A vSphere 7, vSphere 6.7u3 or later, VMware Cloud on AWS, or Azure VMware Solution account with
```

Comment:
I'd suggest replacing `account` with `environment` since that is what TCE is being deployed to.

Reply:
thanks - done